engine/api/user.go

```diff
@@ -50,6 +54,10 @@ func GetUserHandler(w http.ResponseWriter, r *http.Request, db *gorp.DbMap, c *c
 	vars := mux.Vars(r)
 	username := vars["name"]
 
+	if !c.User.Admin && username != c.User.Username {
```

Comment:
Maybe it should be wrapped in a NeedUser() func on router.Handle to avoid duplicating this security permission